tools/CoqMakefile.in

```diff
@@ -783,22 +783,48 @@ $(filter-out $(MLLIBFILES:.mllib=.cmxs) $(MLPACKFILES:.mlpack=.cmxs) $(addsuffix
 		-shared -o $@ $<
 
 ifneq (,$(TIMING))
-TIMING_EXTRA = > $<.$(TIMING_EXT)
+TIMING_EXTRA = > $(1).v.$(TIMING_EXT)
 else
 TIMING_EXTRA =
 endif
 
+# can't make
+# https://www.gnu.org/software/make/manual/make.html#Static-Pattern
+# work with multiple target rules
+# so use eval in a loop instead
+# with grouped targets https://www.gnu.org/software/make/manual/make.html#Multiple-Targets
+# if available (GNU Make >= 4.3)
+ifneq (,$(filter grouped-target,$(.FEATURES)))
+define globvorule=
+
+# delayed variables like $< don't work right in the eval so we have to use $(1) instead
```

Comment:
That's not true, you just need to double the dollar signs to delay evaluation to rule execution time